### ISSUE TITLE
No Pseudo mods

### ISSUE THREAD

**Issue by Kvan7** [Kvan7/Exiled-Exchange-2#51]
The trade site doesn't support pseudo mods which means most stuff is a lot worse to filter by now it seems?

**Comment by Bennnsen** [comment]
Yeah. If you currently search for explicit fire res, the search won´t show rings with fire res implicit. Not sure how the requests work, but if you just want fire res on rings, best to do is currently with the weighted sum. Not sure if that´s even possible. Sounds too complicated to invest hours of work at the start. Would also just wait for them to add pseudo.

**Comment by Kvan7** [comment]
yeah weighted sum works well. it just also eats 12 (9 without runes) of the mods, whereas total res is 1. :) for now im just going to ignore this problem probably until the parser is working

![image](https://github.com/user-attachments/assets/7a1421d9-cba0-41f3-b46c-4e14416b6e6b)

**Comment by Bennnsen** [comment]
Don´t understand. Each entry has equal weight, whereas overall you want min 25 res. Guess that could be a bonus if everything else is done. Not having to put in every single entry manually is already a huge help. Thanks for your effort in fixing it for PoE2!

**Comment by ThirteenEra** [comment]
For now as a duct tape fix can we make the app ignore the enchant/rune mods and focus only on explicit/implicits? This would give it a lot more useability without adding too much headache

**Comment by Kvan7** [comment]
I mean, that would help to solve the limited filter slots with the trade site, but that isn't the biggest part of this, main thing would be changing the logic surrounding pseudo mods so that they use the weighted sums instead

**Comment by Kvan7** [comment]
> This will add complexity, but this 2 is pretty common on rings to count total pseudo res

Are these not on it? if not, uh oops. I'll fix that for next release then

> But for some items I don't see the "Hidden" button:

I'm not sure why hidden isn't showing up for these. I can reprod this so I'll look into it

> Also the "Mods" switch doesn't seem to work (I don't remember what it's supposed to do, haven't played PoE1 few years)

I tbh never used it so I'm not sure what it does either. 
Just looked into it, it is supposed to show the "sources" of pseudo mods/mods that have multiple things feeding into them. This is not working due to my hacky  implementation of pseudo-pseudos since they aren't actual filters on the trade site.

**Comment by DmitryKovarniy** [comment]
> There are no `pseudo` modifiers for stat filters on the trade site yet. this makes price checks much more complicated. For the **exact right now time** I am going to play [ostrich](https://en.wikipedia.org/wiki/Ostrich_algorithm) and deal with this later.
Hello, not sure if a separate issue report is needed, but corruptions are now counted as 'enhance.' If you want to price-check a corrupted item with the Resistance implicit when 'pseudo' is enabled, you are redirected to the trade site with "~imp" resistance filters, and not '~ench' ones, as output you get no results because of it.
https://www.pathofexile.com/trade2/search/poe2/Standard/8D5RywBFV

**Comment by Kvan7** [comment]
> Pseudo search do not work for corruption implicit

yeah, in all honesty I did remove it from the filters manually. With the current filters the trade site `complexity` is around 95-105 on standard queries, when i include runes and enchants, the complexity bumps up to 125-135. Just for my own sanity and trying to limit where things break, I chose to not include it since it raises the complexity and at that time, the complexity was where things were breaking the most. In theory we get pseudo mods actually implemented and I can take out all this hacky stuff.
